### PR TITLE
📖  add title icons list to PR template

### PR DESCRIPTION
**Pull request by nader-ziada**
**What this PR does / why we need it**:
- add description of which title icons to use to PR template

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**TODOs**:

- [x] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
Add title icons list to PR template
```

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
  <!-- If this is your first PR, welcome! Please make sure you read the [contributing guidelines](../CONTRIBUTING.md). -->
 
+ <!-- please add a icon to the title of this PR (see ../CONTRIBUTING.md), and delete this line and similar ones -->
```
Comment (CecileRobertMichon) [inline .github/pull_request_template.md:3]:
```suggestion
 <!-- please add an icon to the title of this PR (see ../CONTRIBUTING.md), and delete this line and similar ones -->
```

Comment (CecileRobertMichon) [reply .github/pull_request_template.md:3]:
```suggestion
 <!-- please add an icon to the title of this PR (see ../CONTRIBUTING.md) -->
```
(?)
why delete the comment lines?

Comment (nader-ziada) [reply .github/pull_request_template.md:3]: it says that in CAPI :/

**Review by CecileRobertMichon — APPROVED** [review]
/lgtm
/approve

**Pull request merged**